### ISSUE TITLE
[Feature Request] Standalone miss command line arguments

### ISSUE THREAD

**Issue by FlachyJoe** [timothyschoen/PlugData#27]
Hi,
I'd like to use command line arguments as original pure-data offers. At least
```
-path <path>     -- add to file search path
-helppath <path> -- add to help file search path
-open <file>     -- open file(s) on startup
-send "msg..."   -- send a message at startup, after patches are loaded
```

**Comment by timothyschoen** [comment]
Good idea! I have to focus on fixing some bugs for now, but I'll get to this soon

**Comment by timothyschoen** [comment]
Opening from commandline should now work, without the -open flag. I'm going to implement -path, -helppath and -send in a later release.

**Comment by timothyschoen** [comment]
Pd has tons of command line arguments, I've added all the ones that are important or easy to implement. The others will be implemented at some point. This is what I have now:

"-nostdpath"
"-stdpath"
"-open"
"-lib"
"-verbose"
"-noverbose"
"-stderr"
"-nostderr"
"-noloadbang"
"-loadbang"
"-send"
"-sleepgrain"
"-compatibility"

Unfortunately helppath still needs to be implemented. Some of there might not work, I took the implementation from pd but haven't found time to test them all.
